lib/sms.rb

```diff
@@ -0,0 +1,39 @@
+require "twilio-ruby"
+
+class SMS
+
+	TIME_FORMAT = "%H:%M"
+
+	def initialize(config, client:Twilio::REST::Client.new(config[:account_sid], config[:auth_token]))
```

Comment:
I would probably store the account_sid & the auth_token in environment variables.